thinc/backends/_custom_kernels.py

```diff
@@ -146,6 +157,61 @@ def _alloc_like(array, zeros: bool = True):
         return cupy.empty_like(array)
 
 
+def pad(seqs, round_to=1, *, threads_per_block=128, num_blocks=128):
+    for seq in seqs:
+        _is_float_or_int_array(seq)
+
+    lengths = [len(seq) for seq in seqs]
+    length = max(lengths)
```

Comment:
Can we rename these two to `seq_lens` and `max_seq_len` (or something similar) to avoid confusion?

Reply:
Done.